miner/worker.go

```diff
@@ -165,10 +165,6 @@ func (miner *Miner) prepareWork(genParams *generateParams, witness bool) (*envir
 				return nil, fmt.Errorf("invalid timestamp, parent %d given %d", parent.Time, timestamp)
 			}
 			timestamp = parent.Time + 1
-		} else {
-			if parent.Time > timestamp {
```

Comment:
If `parent.Time > timestamp` won't return error, is this expected?

Reply:
Yes, soft block is a preconfed block in future

Reply:
Added back